ramp-frontend/ramp_frontend/templates/update_event.html

```diff
@@ -37,15 +37,23 @@ <h4 class="modal-title" id="myModalLabel">{{ category }}</h4>
         </div>                   
          <div class="card-body">
            <form class="ui form" method="post" action="" name="update_event">
-               {{ form.hidden_tag() }}
-                   Event name will be {{ event.problem.name }} if you leave suffix empty, or {{ event.problem.name }}_&lt;suffix&gt;. The length should be less than 20 characters, only ascii, no space. It will be used to identify the event, primarily in the url. If you open several events for the same class or hackaton, please use the same suffix consistently.
-                   <div class="form-group">
-                       <label>event suffix</label>
-                       {{ form.suffix(placeholder="event_suffix") }}
-                       {% for error in form.suffix.errors %}
-                           <span style="color: red;">[{{ error }}]</span>
-                       {% endfor %}
+                {% if not suffix_correct %}
+                    <div class="form-group"></div>
+                        <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+                        Event name does not correspond to the RAMP standards. In the future, please make sure that it is preceeded by the 'problem_name_', eg: '{{ event.problem.name }}_{{ event.name }}'
```

Comment:
Do we actually care to warn. Basically people will get a ValueError when deploying. I would just remove the field and that's it.